sandpack-react/src/components/Tests/utils.ts

```diff
@@ -83,3 +83,60 @@ export const set =
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
   <A extends Record<string, any>>(object: A): A =>
     cleanSet(object, path, value);
+
+export class ReactStringReplacer {
```

Comment:
Could you leave a comment here explaining the reason for using this implementation and what it does?